### PR TITLE
KEP: Work Placement

### PR DESCRIPTION
**Pull request by jnpacker**
Signed-off-by: Joshua Packer <jpacker@redhat.com>
* Initial submission for Work placement enhancement

### CODE REVIEW

File: enhancements/sig-architecture/35-work-placement/work-placement.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+          namespace: default
+        spec: … 
+      - apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: test02
+          namespace: default
+        spec: … 
+  ## The following is a spike ##
+  distributionStrategy:
```
Comment (serngawy) [inline enhancements/sig-architecture/35-work-placement/work-placement.md:90]: can we elaborate more on the distributionStrategy ? what is the type Split ? and why is the distributionStrategy is repeated under test02 item as the Type is split ?

**Comment by ivan-cai** [comment]
Should we need different configuration across clusters? For example, we want to distribute some objects like ResourceQuota/LimitRanger to multiple clusters，but the object value is different in different clusters.

**Comment by jnpacker** [comment]
That could definitely be a spike after the initial merge.  The API is done I will submit the PR this week. I hopefully will also port the controller to the ManifestWork repo (currently in its own) this week as well to get some feedback.

File: enhancements/sig-architecture/35-work-placement/work-placement.md (hunk 1 above)
Comment (jnpacker) [reply enhancements/sig-architecture/35-work-placement/work-placement.md:90]: Once we have the base controller, we can start working on the strategy spike.

**Pull request merged**